client/my-sites/backup/backup-contents-page/file-browser/file-browser-node.tsx

```diff
@@ -16,6 +16,7 @@ interface FileBrowserNodeProps {
 	isAlternate: boolean; // This decides if the node will have a background color or not
 	setActiveNodePath: ( path: string ) => void;
 	activeNodePath: string;
+	parentItem?: FileBrowserItem; // This is used to pass the extension details to the child node
```

Comment:
This is required because details like the `extension_slug` and `extension_version` are available in the parent node. In the following image, the `akismet` is the parent node.

![image](https://github.com/Automattic/wp-calypso/assets/1488641/e7d77add-cbe1-4ec1-98be-cf9e14b391e5)